tests/ftests/088-sudo-cgclassify_systemd_scope.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Test to replace idle_thread of scope unit using cgclassify -r
+#
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from process import Process
+from cgroup import Cgroup
+from run import RunError
+import consts
+import ftests
+import time
+import sys
+import os
+
+CONTROLLER = 'cpu'
+SLICE = 'libcgroup.slice'
+CGNAME = os.path.join(SLICE, '086cgcreate.scope')
+OUT_OF_SCOPE_CGNAME1 = '086outofscope'
+OUT_OF_SCOPE_CGNAME2 = '086outofscope.scope'
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if config.args.container:
+        result = consts.TEST_SKIPPED
+        cause = 'This test cannot be run within a container'
+
+    return result, cause
+
+
+def setup(config):
+    pass
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    #
+    # Test 1: call cgclassify -r to replace idle_thread with infinite while loop
+    #
+    Cgroup.create_and_validate(config, CONTROLLER, CGNAME, create_scope=True)
+    idle_pid = Cgroup.get_pids_in_cgroup(config, CGNAME, CONTROLLER)[0]
+
+    config.process.create_process_in_cgroup(config, CONTROLLER, CGNAME, replace_idle=True)
```

Comment:
Since this test is designed to test cgclassify, you may want to explicitly specify that (in case the default ever changes):
```
config.process.create_process_in_cgroup(config, CONTROLLER, CGNAME, cgclassify=True, replace_idle=True)
```

Reply:
Good idea will add `cgclassify=True` to the argument list.